platforms/documentation/docs/src/docs/userguide/dep-man/dependency-management/component_metadata_rules.adoc

```diff
@@ -283,13 +251,13 @@ In the build script, we can now request a specific variant and Gradle will fail
 Gradle is able to understand the common case where a single attribute is missing that would have removed the ambiguity.
 In this case, rather than listing information about all attributes on all available variants, Gradle helpfully lists only possible values for that attribute along with the variants each value would select.
 
-.Applying and utilising rule for LWGJ metadata
 ====
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/kotlin",files="build.gradle.kts[tags=lwgj-dependencies]"]
 include::sample[dir="snippets/dependencyManagement/customizingResolution-metadataRule/groovy",files="build.gradle[tags=lwgj-dependencies]"]
 ====
 
-.Gradle fails to select a variant because a machine architecture needs to be chosen
+Gradle fails to select a variant because a machine architecture needs to be chosen:
+
```

Comment:
"Because it is difficult to model <<feature_variants.adoc#feature_variants,optional feature variants>> as separate jars with pom metadata, libraries sometimes compose different jars with a different feature set" below should maybe say: "Because it is difficult to model <<feature_variants.adoc#feature_variants,optional feature variants>> as separate jars with pom metadata, libraries sometimes COMPRISE different jars with different feature setS"

Reply:
Fixed. TY.